tools/process_symbol_map.py

```diff
@@ -358,6 +358,18 @@ def generate_init(self, out: Output):
             for ver, symbols in self.symbol_dict.items():
                 if ver == "default":
                     out.cxx("\telse {")
+                elif ">>>" in ver:
```

Comment:
While I appreciate the PR this approach will not work, for example the else statement executes and puts an if version == 1.14.60.5, this would cause a range that has that version to not be run due to the if statement skipping it.
A better approach would be to do these ranges before this loop as to not break the "default" implementation and not to use else clauses, also I think this should just be simplified to ver - ver using the >>> implementation.